olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -300,12 +317,23 @@ bool NetworkCurl::Initialize() {
     return false;
   }
 
+#if defined(OLP_SDK_ENABLE_ANDROID_CURL) && defined(OLP_SDK_NETWORK_HAS_OPENSSL)
+  md5_lookup_method_ = X509_LOOKUP_meth_new(kLookupMethodName);
+
+  X509_LOOKUP_meth_set_ctrl(md5_lookup_method_, Md5LookupCtrl);
+  X509_LOOKUP_meth_set_get_by_subject(md5_lookup_method_,
+                                      Md5LookupGetBySubject);
```

Comment:
Please add comment why we do this. Describe the issue and what we are doing to mitigate.